src/strings.ts

```diff
@@ -0,0 +1,2 @@
+export const TALKER_TYPE: string = 'application/x-talkerjs-v1+json';
```

Comment:
Nit: content type is a bit more descriptive.
Nit: you could just call this file `constants.js` for more future flexibility with it.

```suggestion
export const TALKER_CONTENT_TYPE: string = 'application/x-talkerjs-v1+json';
```